ci-operator/step-registry/cucushift/hypershift-extended/calico/health-check/cucushift-hypershift-extended-calico-health-check-ref.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/hypershift-extended/calico/health-check/cucushift-hypershift-extended-calico-health-check-ref.yaml",
+	"owners": {
+		"approvers": [
+			"LiangquanLi930",
+			"heliubj18",
+			"fxierh"
```

Comment:
Could you help to remove 'fxierh', he has left Red Hat

Reply:
OK. Let me fix it.

Reply:
Removed the user from my code (pushed a new commit). And I will send a separate PR to remove the user from all folders in the repo.